forch/forchestrator.py

```diff
@@ -214,17 +225,26 @@ def _register_handlers(self):
                 event.timestamp, event.dp_name, event.port_no, event.eth_src, event.l3_src_ip)),
         ])
 
+    def _get_varz_config(self):
+        metrics = self._varz_collector.get_metrics()
+        varz_hash_info = metrics['faucet_config_hash_info']
+        assert len(varz_hash_info.samples) == 1, 'exactly one config hash info not found'
+        varz_config_hashes = varz_hash_info.samples[0].labels['hashes']
+        varz_config_error = varz_hash_info.samples[0].labels['error']
+
+        if varz_config_error:
+            raise Exception(f'Varz config error: {varz_config_error}')
+
+        return metrics, varz_config_hashes, varz_config_error
```

Comment:
no need to return varz_config_error -- it will always be None

Reply:
Done